src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -0,0 +1,43 @@
+package uk.gov.nationalarchives
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent
+import com.dimafeng.testcontainers.DynaliteContainer
+import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import uk.gov.nationalarchives.referencegenerator.Lambda
+import uk.gov.nationalarchives.utils.TestContainerUtils
+
+class LambdaTest extends AnyFlatSpec with Matchers with TestContainerUtils {
+  override def config: Config = ConfigFactory.load()
+
+  override def afterContainersStart(containers: containerDef.Container): Unit = super.afterContainersStart(containers)
+
```

Comment:
Should probably add a test for when the query param is not an `Int`

Reply:
Tried adding a test for this. But Scala being a static language won't let me convert different types to an `Int`. I have however added some logic in the `Lambda.scala` to handle this case.